### ISSUE TITLE
[Bug]: 'Error loading file picker template: {error}' when move/copy files after update to 26.0.2

### ISSUE THREAD

**Issue by jacotec** [nextcloud/server#38500]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

After update to 26.0.2 I always get the error popup with the mentioned error after selecting "Move or copy" for a file:

<img width="439" alt="2023-05-29_13-11-11" src="https://github.com/nextcloud/server/assets/3321655/1cb58dac-4c77-4104-b450-af15b241d416">

After clicking "OK" the file selection window opens and I can move the file, but the window does not close after the action.

### Steps to reproduce

1. Open folder with files in the browser
2. Click on "..." and select "move or copy"
3. The error message appears

### Expected behavior

The file picker should show up immediately without error message

### Installation method

Community Manual installation with Archive

### Nextcloud Server version

26

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database engine version

MySQL

### Is this bug present after an update or on a fresh install?

Updated from a minor version (ex. 22.2.3 to 22.2.4)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [X] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "10.0.1.6",
            "***REMOVED SENSITIVE VALUE***"
        ],
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "forwarded_for_headers": [
            "HTTP_X_FORWARDED_FOR",
            "HTTP_FORWARDED_FOR"
        ],
        "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
        "overwriteprotocol": "https",
        "allow_local_remote_servers": true,
        "overwritehost": "***REMOVED SENSITIVE VALUE***",
        "htaccess.RewriteBase": "\/",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "26.0.2.1",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "maintenance": false,
        "enable_avatars": true,
        "filesystem_check_changes": 1,
        "theme": "",
        "default_language": "de",
        "default_locale": "de_DE",
        "default_phone_region": "DE",
        "defaultapp": "files",
        "skeletondirectory": "\/var\/www\/skeleton",
        "share_folder": "\/Shared",
        "log_type": "file",
        "logfile": "\/var\/log\/nextcloud\/nextcloud.log",
        "loglevel": 2,
        "mail_smtpmode": "smtp",
        "remember_login_cookie_lifetime": 2592000,
        "session_lifetime": 604800,
        "session_keepalive": true,
        "log_rotate_size": "10485760",
        "trashbin_retention_obligation": "30, 60",
        "versions_retention_obligation": "auto, 365",
        "check_for_working_webdav": true,
        "simpleSignUpLink.shown": false,
        "auth.webauthn.enabled": false,
        "music.lastfm_api_key": "2897d80221740bce55bd257a138705d1",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "filelocking.enabled": true,
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "timeout": 0.5,
            "dbindex": 0
        },
        "logtimezone": "Europe\/Berlin",
        "preview_libreoffice_path": "\/usr\/bin\/libreoffice",
        "enable_previews": true,
        "enabledPreviewProviders": [
            "OC\\Preview\\PNG",
            "OC\\Preview\\JPEG",
            "OC\\Preview\\GIF",
            "OC\\Preview\\BMP",
            "OC\\Preview\\MarkDown",
            "OC\\Preview\\MP3",
            "OC\\Preview\\TXT",
            "OC\\Preview\\Movie",
            "OC\\Preview\\Photoshop",
            "OC\\Preview\\SVG",
            "OC\\Preview\\TIFF",
            "OC\\Preview\\HEIC",
            "OC\\Preview\\MKV",
            "OC\\Preview\\MOV",
            "OC\\Preview\\MP4",
            "OC\\Preview\\AVI",
            "OC\\Preview\\PDF",
            "OC\\Preview\\OpenDocument",
            "OC\\Preview\\MSOffice2003",
            "OC\\Preview\\MSOffice2007",
            "OC\\Preview\\MSOfficeDoc",
            "OC\\Preview\\Image",
            "OC\\Preview\\PPT",
            "OC\\Preview\\PPTX"
        ],
        "preview_max_x": "2048",
        "preview_max_y": "2048",
        "jpeg_quality": "60",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_sendmailmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "587",
        "ldapIgnoreNamingRules": false,
        "ldapProviderFactory": "OCA\\User_LDAP\\LDAPProviderFactory",
        "ldapUserCleanupInterval": 51,
        "app_install_overwrite": [
            "files_external_dropbox",
            "bookmarks_fulltextsearch",
            "files_inotify",
            "files_fulltextsearch_tesseract",
            "previewgenerator",
            "quickaccesssorting",
            "bbb",
            "fulltextsearch",
            "fulltextsearch_elasticsearch",
            "files_fulltextsearch",
            "socialsharing_email",
            "extract",
            "drawio",
            "quicknotes",
            "video_converter",
            "oidc_login",
            "facerecognition",
            "files_linkeditor",
            "metadata",
            "cfg_share_links",
            "ncdownloader"
        ],
        "updater.release.channel": "stable",
        "mail_smtpauth": 1,
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpsecure": "tls",
        "mail_smtpauthtype": "LOGIN",
        "oidc_login_provider_url": "***REMOVED SENSITIVE VALUE***",
        "oidc_login_client_id": "***REMOVED SENSITIVE VALUE***",
        "oidc_login_client_secret": "***REMOVED SENSITIVE VALUE***",
        "oidc_login_end_session_redirect": true,
        "oidc_login_logout_url": "***REMOVED SENSITIVE VALUE***",
        "oidc_login_auto_redirect": false,
        "oidc_login_hide_password_form": false,
        "oidc_login_attributes": {
            "id": "preferred_username",
            "name": "name",
            "mail": "email",
            "quota": "nextcloudquota",
            "groups": "groups"
        },
        "oidc_login_scope": "openid profile",
        "oidc_login_disable_registration": false,
        "oidc_login_redir_fallback": true,
        "oidc_login_password_authentication": true,
        "oidc_login_button_text": "Mit FLACCS Cloud anmelden",
        "memories.exiftool": "\/var\/www\/nextcloud\/apps\/memories\/lib\/..\/exiftool-bin\/exiftool-amd64-glibc",
        "memories.gis_type": 1,
        "enabledFaceRecognitionMimetype": [
            "image\/jpeg",
            "image\/png",
            "image\/heic",
            "image\/tiff",
            "image\/webp"
        ],
        "memories.vod.ffmpeg": "\/usr\/bin\/ffmpeg",
        "memories.vod.ffprobe": "\/usr\/bin\/ffprobe",
        "memories.vod.path": "\/var\/www\/nextcloud\/apps\/memories\/exiftool-bin\/go-vod-amd64",
        "memories.vod.disable": false,
        "memories.vod.tempdir": "\/tmp\/memories_cache"
    }
}
```

### List of activated Apps

```shell
Enabled:
  - activity: 2.18.0
  - admin_audit: 1.16.0
  - appointments: 1.15.1
  - bbb: 2.4.0
  - bookmarks: 13.0.1
  - bruteforcesettings: 2.6.0
  - calendar: 4.3.4
  - camerarawpreviews: 0.8.1
  - cfg_share_links: 3.0.2
  - circles: 26.0.0
  - cloud_federation_api: 1.9.0
  - comments: 1.16.0
  - contacts: 5.2.0
  - contactsinteraction: 1.7.0
  - dashboard: 7.6.0
  - dav: 1.25.0
  - deck: 1.9.2
  - drawio: 2.1.1
  - extract: 1.3.5
  - facerecognition: 0.9.12
  - federatedfilesharing: 1.16.0
  - federation: 1.16.0
  - files: 1.21.1
  - files_accesscontrol: 1.16.0
  - files_automatedtagging: 1.16.1
  - files_downloadactivity: 1.16.0
  - files_external: 1.18.0
  - files_fulltextsearch: 26.0.0
  - files_fulltextsearch_tesseract: 25.0.0
  - files_inotify: 0.1.14
  - files_linkeditor: 1.1.14
  - files_pdfviewer: 2.7.0
  - files_retention: 1.15.0
  - files_rightclick: 1.5.0
  - files_sharing: 1.18.0
  - files_trashbin: 1.16.0
  - files_versions: 1.19.1
  - firstrunwizard: 2.15.0
  - forms: 3.3.0
  - fulltextsearch: 26.0.0
  - fulltextsearch_elasticsearch: 26.0.0
  - groupfolders: 14.0.2
  - guests: 2.5.0
  - integration_openai: 1.0.8
  - logreader: 2.11.0
  - lookup_server_connector: 1.14.0
  - maps: 1.0.2
  - memories: 5.1.0
  - metadata: 0.18.0
  - news: 21.2.0
  - nextcloud_announcements: 1.15.0
  - notes: 4.7.2
  - notifications: 2.14.0
  - notify_push: 0.6.3
  - oauth2: 1.14.0
  - oidc_login: 2.5.1
  - photos: 2.2.0
  - polls: 5.0.5
  - previewgenerator: 5.3.0
  - privacy: 1.10.0
  - provisioning_api: 1.16.0
  - qownnotesapi: 23.3.0
  - quickaccesssorting: 3.0.0
  - quicknotes: 0.8.5
  - quota_warning: 1.17.0
  - recommendations: 1.5.0
  - richdocuments: 8.0.2
  - serverinfo: 1.16.0
  - settings: 1.8.0
  - sharebymail: 1.16.0
  - side_menu: 3.8.0
  - socialsharing_email: 2.6.0
  - spreed: 16.0.4
  - support: 1.9.0
  - survey_client: 1.14.0
  - systemtags: 1.16.0
  - tasks: 0.15.0
  - text: 3.7.2
  - theming: 2.1.1
  - translate: 1.1.3
  - twofactor_backupcodes: 1.15.0
  - twofactor_totp: 8.0.0
  - updatenotification: 1.16.0
  - user_ldap: 1.16.0
  - user_status: 1.6.0
  - video_converter: 1.0.5
  - viewer: 1.10.0
  - weather_status: 1.6.0
  - workflow_pdf_converter: 1.11.0
  - workflowengine: 2.8.0
Disabled:
  - audioplayer: 3.3.1 (installed 3.3.1)
  - encryption: 2.14.0
  - music: 1.8.3 (installed 1.8.3)
  - ncdownloader: 1.0.7 (installed 1.0.7)
  - ocdownloader: 1.9.1 (installed 1.8.1)
  - password_policy: 1.16.0 (installed 1.15.0)
  - ransomware_protection: 1.14.0 (installed 1.14.0)
  - related_resources: 1.1.0-alpha1 (installed 1.1.0-alpha1)
  - suspicious_login: 4.4.0
  - talk_matterbridge: 1.26.0-0 (installed 1.26.0-0)
  - workflow_script: 1.11.1 (installed 1.11.1)
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

```shell
Issue produces no log entries
```

### Additional info

The last entry in the browser console before the error popoup is:

```
e.indexOf is not a function TypeError: e.indexOf is not a function
    at https://mydomain.de/dist
    
jquery.js:4050

```

**Comment by jacotec** [comment]
Disabling the app "Configurable share links" (cfg_share_links) eliminates the error - seems to occur only when this app is active.

**Comment by jacotec** [comment]
@szaimen Why did you close the issue? It's not solved and was introduced by a NC update (26.0.1 --> 26.0.2), nothing else was changed.

**Comment by szaimen** [comment]
Needs to be addressed in https://github.com/jimmyl0l3c/cfg_share_links/issues/143
